KubeArmor/feeder/policyMatcher.go

```diff
@@ -1080,6 +1080,10 @@ func (fd *Feeder) UpdateMatchedPolicy(log tp.Log) tp.Log {
 						matchedRegex = strings.HasSuffix(log.ProcessName, "/"+secPolicy.Resource) // processpath = */execname
 					}
 
+					// fallback logic: in case we get relative path in log.Resource then we join cwd + resource to get pull path
+					if !strings.HasPrefix(strings.Split(log.Resource, " ")[0], "/") && log.Cwd != "/" {
```

Comment:
Do you think there will be a case where, 
Cwd[0] != "/"?
That is we have a relative path in CWD.

Reply:
I don't think we can get relative path in cwd